caffe2/python/onnx/backend.py

```diff
@@ -205,7 +205,7 @@ class Caffe2Backend(Backend):
     # so this interface MAY make BC-breaking changes.  Specify an
     # opset_version if you don't want this to version.
     @classmethod
-    def run_node(cls, node, inputs, device='CPU', opset_version=_known_opset_version):
+    def run_node(cls, node, inputs, device='CPU', outputs_info=None, opset_version=_known_opset_version):
```

Comment:
new arg should be put at the end (in case there are callers invoking this function with all positional args).

Reply:
onnx backend test suit should pass this new arg as keyword arg.

Reply:
Sure, that sounds good to me. Just switched the order.